sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -53,6 +53,70 @@
     :background-color="hasDisbursementEvent ? undefined : 'success-bg'"
     content="StudentAid BC has determined an outcome with 1 or more of your requested change. Please review your new assessment in the table below."
   />
+  <!-- Student application change request - in progress with student -->
+  <application-status-tracker-banner
```

Comment:
I believe after scholastic standing, the order of card arrangements must be based on status.

If we see the the in-progress component we arrange all the denial cards and then the waiting cards. 

I believe here, we must put all the in-progress status  cards(after scholastic standing cards), and then follow the same existing order for other status.

@ann-aot @andrewsignori-aot @guru-aot  thoughts?

Reply:
I see the point but would it not possibly mess with the order that they are displayed?
For instance, for "student appeals" and "offering change request" we can have a complete card and also an in-progress card displayed at the same time. In this way, they are displayed as a group. Does it make sense?

Reply:
I actually, didn't get the qn

Reply:
As the Team agreed, let's have them organized as below:
- Anything In progress
- Anything declined
- Anything trigger type based
- Disbursement/COE banners